doc/reflection-milestone4.md

```diff
@@ -0,0 +1,40 @@
+# Reflection
+
+## What has been implemented
+
+The python dashboard features 7 plots which describe the distribution of data science salaries for the features country, gender, education level, age, tenure, and a scatter plot of all the data science salaries. The dashboard is split into two parts: the left side shows country-specific salary distributions while the right side (marked by a darker background) shows the global salary trends. The global salary trends are colored based on the data scientist's tenure, and interactivity is provided to filter based on tenure in the python dashboard. However, this feature was removed in the R dashboard due to incompatibility of the two plots in plotly. Both functions are fully functional and have been implemented in Heroku.
```

Comment:
I think the last sentence can be removed, or reworded to say that the dashboards are deployed to Heroku.